tests/unit/cache/test_schema_cache.py

```
@@ -293,3 +293,145 @@ def test_parse_unknown_field_type(strapi_config):
+
+        schema = cache._parse_schema_response("api::article.article", schema_data)
+
+        # Should correctly handle nested schema and extract flat info
+        assert schema.display_name == "Article"
+        assert schema.singular_name == "article"
+        assert schema.plural_name == "articles"
+        assert schema.kind == "collectionType"
+
+
+@respx.mock
+def test_extract_info_from_schema_flat(strapi_config):
```

Ruff (0.14.14)

[warning] Unused function argument: `strapi_config`

(ARG001)